app/views/ansible_credential/edit.html.haml

```diff
@@ -1,2 +1,3 @@
 #main_div
   = render :partial => 'credential_form', :locals => {:newRecord => false}
+  = react('AnsibleCredentialsForm')
```

Comment:
To distinguis between an edited entity and a new one, we should pass here the entity ID as an additional attribute:
```haml
= react('AnsibleCredentialsForm', :recordId => @id)
```